aiven_mysql_migrate/main.py

```diff
@@ -62,6 +62,13 @@ def main(args: Sequence[str] | None = None, *, app: str = "mysql_migrate") -> Op
         default=None,
         help="Output file which includes metadata such as dump GTIDs (for replication method only) in JSON format.",
     )
+    parser.add_argument(
+        "--migration-error-path",
```

Comment:
Consistency: The previous argument is called output-something-file, this one something-error-path. I'd like at least for path to be changed to file instead. Whether the word "migration" is even needed,,, I could almost see this being simply --error-file?